app/_locales/en/messages.json

```diff
@@ -3043,11 +3043,14 @@
     "message": "Install Snap"
   },
   "snapInstallWarningCheck": {
-    "message": "To confirm you understand, check all."
+    "message": "To confirm you understand, check the box."
+  },
+  "snapInstallWarningCheckPlural": {
+    "message": "To confirm that you understand, check the box."
```

Comment:
Ì think you might have had a regression in the copy here with all the rebases?

Reply:
Otherwise this should be good to go finally.

Reply:
Fixed :) This might have regressed because of the merge fail I had couple of days ago